### ISSUE TITLE
Syntax highlighting not done for else and endif

### ISSUE THREAD

**Issue by comiluv** [vigoux/tree-sitter-viml#21]
**Describe the bug**

Syntax highlighter does not highlight else and endif in some cases.

**To Reproduce**

1. Make a viml file: test-vim.vim
2. Type following
```viml
" WSL yank support
let s:clip = '/mnt/c/Windows/System32/clip.exe'  " default location
if has('unix') && !has('nvim') && executable(s:clip) && has('patch8.0.1394')
    augroup WSLYank
        autocmd!
        autocmd TextYankPost * call system('echo '.shellescape(join(v:event.regcontents, "\<CR>")).' | '.s:clip)
    augroup END
    noremap "+p :exe 'norm a'.system('/mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -Command Get-Clipboard')<CR>
endif

" C Compile settings. F5 to compile with GCC and F8 to execute
let compile_cmdline='gcc % -g -Wall -Wextra -pedantic -o %<'
if has('nvim')
    nnoremap <F5> :w! <BAR> execute 'terminal '.compile_cmdline<CR>
else
    nnoremap <F5> :w! <BAR> execute '!'.compile_cmdline<CR>
endif
if !has("unix")
    if has('nvim')
        noremap <F8> :terminal %<<Enter>
    else
        noremap <F8> :!%<<Enter>
    endif
elseif has("unix")
    if has('nvim')
        noremap <F8> :terminal ./%<<Enter>
    else
        noremap <F8> :!./%<<Enter>
    endif
endif

" loop through subdirectory and source .vim files
" https://johngrib.github.io/wiki/vim-configure-split/
for include_file in uniq(sort(globpath(&rtp, 'vim-include/*.vim', 0, 1)))
    execute "source " . include_file
endfor
```
3. Some else, endif statements are not highlighted

See lines 97, 103, 105, 109, 115 in this screenshot
<img width="831" alt="화면 캡처 2021-08-23 133900" src="https://user-images.githubusercontent.com/19642705/130391144-41d533cb-65db-4d26-b4e1-312af047fbcc.png">

**Expected behavior**

All else and endif statements should be highlighted.

**Comment by comiluv** [comment]
Moved from nvim-treesitter/nvim-treesitter#1732

**Comment by vigoux** [comment]
Hi could you provide the text used for this example ?

**Comment by vigoux** [comment]
From what I see, this may have been fixed by b37f5ac645d3db1f5ef9c9b07f12d760fadc038d, could you confirm it is the case ?

**Comment by comiluv** [comment]
Last update seems fixed this issue. Highlighting in above example looks all correct after updating. Thank you
